tensorflow/lite/delegates/gpu/common/shape.h

```diff
@@ -362,7 +362,9 @@ struct DimensionGetterFunc {
   template <Layout T>
   int32_t operator()() const {
     int i = GetAxisIndex<T>(d);
-    return i >= 0 && i < l->dimensions.size() ? l->dimensions[i] : -1;
+    return i >= 0 && static_cast<size_t>(i) < l->dimensions.size()
```

Comment:
rather than casting i, i think it's more common to cast the rhs.  here and in other places too